2018/g2018_vr.csv

```diff
@@ -4,7 +4,7 @@ AK,2018-10-07,2018-10-07,,,
 AZ,2018-10-09,2018-10-09,,,
 AR,2018-10-07,,,,
 CA,2018-10-22,2018-10-22,,,
-CO,,,,,Voters can register at a polling center up to and including Election Day.
+CO,2018-11-06,,,,Voters can register at a polling center up to and including Election Day.
```

Comment:
There's been some conversation about same-day registration. For example, I had opened a Colorado PR (#4) that was closed. @fpereyra has said:
>we prefer to leave the dates blank for states with no real voter registration deadlines and instead include a note to further explain

I've also opened an Issue (#31) asking how to handle states that have no real in-person deadline but do have online deadlines.